sfdo_template_helpers/fields/markdown.py

```diff
@@ -28,14 +24,14 @@ def __get__(self, instance, owner):
         raw_md = instance.__dict__[self.field.name]
         if raw_md is None:
             return ""
-        safer = mark_safe if self.field.is_safe else supplier
```

Comment:
😅 this seemed like a good idea at the time, so thank you for making it actually understandable :)